lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -1378,13 +1378,37 @@ class LineChartPainter extends AxisChartPainter<LineChartData> {
     /// draw the texts one by one in below of each other
     var topPosSeek = tooltipData.tooltipPadding.top;
     for (var tp in drawingTextPainters) {
-      final drawOffset = Offset(
-        rect.center.dx - (tp.width / 2),
-        rect.topCenter.dy +
-            topPosSeek -
-            textRotationOffset.dy +
-            rectRotationOffset.dy,
-      );
+      double yOffset = rect.topCenter.dy +
```

Comment:
@prajwal27 I think we can apply it simpler. We need to add `minWidth` property to force the text to use our defined with. Then `textDirection` should work correctly.

Check what we did for side titles:
https://github.com/imaNNeoFighT/fl_chart/blob/711c937855aa3d62732c631a658457680e4c7662/lib/src/chart/line_chart/line_chart_painter.dart#L927-L930

It would be great if you fix it for wherever we have tooltip (`line_chart_painter`, `bar_chart_painter` and `scatter_chart_painter` at the moment)

Reply:
@imaNNeoFighT  Currently the layout is set like this
```
tp.layout(maxWidth: tooltipData.maxContentWidth);
```

Are you suggesting we do this?
```
tp.layout(
        maxWidth: tooltipData.maxContentWidth,
        minWidth: tooltipData.maxContentWidth,
      );
```

Reply:
What value should we set as `minWidth` ?

Reply:
Yes you are correct!
It makes it `tight` (exact min and max).

Reply:
@imaNNeoFighT , but makes the tooltip's width to be equal to the `maxContentWidth`. Even if the actual content is one character, the tooltip will be a lot bigger. So, I don't think this is the right solution.

Reply:
I checked it. It seems `textAlign` is working correctly. Let me write my answer in the issue.